### PR TITLE
Updates of Xens Race-mods requires changes

### PR DESCRIPTION
**Pull request by emipa606**
## Additions

The new mod-names in all patches

## Changes

Removed redundant patches that already exists in BasePawn

## Reasoning

The race-mods got updated to increase compatibility with mods like Awesome inventory

## Testing

Check tests you have performed:
- [ ] Compiles without warnings
- [x] Game runs without errors
- [ ] (For compatibility patches) ...with and without patched mod loaded
- [ ] Playtested a colony (specify how long)

### CODE REVIEW

File: Patches/Racc/Race_Racc.xml [hunk 1]
```diff
@@ -98,29 +101,6 @@
...
-	</li>
-
-	<li Class="PatchOperationAdd">
-	
-		<xpath>Defs/ThingDef[@Name="BaseRaccPawn"]/comps</xpath>
-		<value>
-			<li Class="CombatExtended.CompProperties_Inventory" />
-		</value>
-	</li>
-		</operations>
```
Comment (N7Huntsman) [inline Patches/Racc/Race_Racc.xml:121]: Patch format is incorrect. You still need this line and the lines below it.

**Review by N7Huntsman — CHANGES_REQUESTED** [review]

File: Patches/Racc/Race_Racc.xml [hunk 2]
```diff
@@ -94,33 +97,11 @@
...
-		</match>
-	</Operation>
```
Comment (N7Huntsman) [inline Patches/Racc/Race_Racc.xml:123]: Very close, but the patch still crashes. I checked, and it just needs this `</Operation>` back.

**Review by N7Huntsman — CHANGES_REQUESTED** [review]
Nearly there. Just one last formatting issue to fix.

File: Patches/Racc/Race_Racc.xml (hunk 2 above)
Comment (emipa606) [reply Patches/Racc/Race_Racc.xml:123]: Should be fixed now, was tired when I wrote it :|

**Review by N7Huntsman — APPROVED** [review]

**Pull request merged**